src/Stratis.StratisNodeWithHtmlUI/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using NBitcoin;
+using NBitcoin.Protocol;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Dashboard;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.StratisNodeWithHtmlUI
```

Comment:
I prefer the name `NodeUI` or `NodeWebUI`
It should probably be `Stratis.StratisUiD` `Stratis.StratisWebUiD`

Reply:
D in StratisD stands for Daemon? Why not use the full word? Or change to "Node"? As a non internal developer, "BitcoinBackgroundDaemon", "StratisBackgroundDaemon", "BreezeBackgroundDaemon", "StratisBackgroundWithWebDashboard" would be my suggested naming for these projects, so that developers who load the main project know what they are.

Perhaps there is a technical reason why you are aiming for a very short name?

The usual external developer workflow is going to be to clone the code, open the first VS solution file they see, and build all. Then they will look for what to run. If the solution explorer lists cryptic short names, they will have to actually find and inspect the Program.cs to understand what StratisD and others each do.

Consider a longer naming scheme for these.

Reply:
Sure I am happy that you rename this daemon with a long name, if it catches we can change the others as well.

Reply:
@StratisDeveloper the -d suffix is just a convention used for Unix like background running processes responsible for "doing the work".
Like in bitcoind, the bitcoin daemon.